about/index.html

```diff
@@ -10,46 +10,45 @@ <h1>About Swedish Forces Pack</h1>
+    The team had no idea how to code or make models, except for Ugga who had previously made some maps for Half-Life.
+    Gradually the four of them started to work towards the goal of getting a squad of swedish soldiers and their weapons into the game.
   </p>
 
   <p>
-    As year passed the team became bigger, got a small fan base and also improved the skills.
-    With the third release of SFP the quality was almost the same as with Operation Flashpoint stuff.
-    The following year the team was invited to the Swedish Army Combat School for taking reference photos and the team grew to around 15 people while the core for most part was 4, with the 3 orignal members and Kungtotte.
+    A year into the project the team had grown and improved their skills, and SFP had gotten a small fan base.
+    With the third release of SFP the quality was almost on par iwth the professionally made content in Operation Flashpoint.
+    The following year the team was invited to take reference photos at the Swedish Army's Land Warfare Centre.
+    At that point the team had grown to around 15 people but still had a 4-man core: the 3 orignal members and Kungtotte.
   </p>
 
   <p>
-    With the forth release for Operation Flashpoint, SFP4.
-    A majority of all weapons used during the last 100 years, plus over 20 vehicles was made and one Swedish island was created with very high detail.
+    With the release of SFP4 for Operation Flashpoint a majority of all weapons used in sweden during the last 100 years had been added. As well as some 20 vehicles and a Swedish island.
   </p>
 
   <p>
-    Most of the team then quit and a community homepage was created, Anrop.se for the Swedish community thats been around SFP for so many years.
-    With the release of ArmA 1 there were some small imports from the OFP material and later new people joined.
-    The Armed Assault of SFP had some new vehicles yet it didn’t become as popular as OFP.
+    Following SFP4 most of the team departed, but a community homepage - <a href="https://anrop.se/" target=_blank>Anrop</a> - was created for the Swedish community that has been around SFP for many years.
+    With the release of ARMA: Armed Assault (Arma 1) came a new version of SFP, with some of the OFP material imported and, of course, additions to the team.
+    For Armed Assault SFP got some new vehicles, but in the end didn’t become quite as popular as OFP.
   </p>
 
   <p>
-    Now the SFP team has a new core, subroc making textures and Steel Rat high quality models.
-    Left with coding is granQ and some of the old SFP members are still around and help out.
-    The aim for SFP is now to limit some aspects like choosing two scenarios, 2015 and 1981 and increase the quality while keeping some of the quantitative that made SFP famous in the first place.
+    For Arma 3 the SFP team has gotten a new core, with subroc making textures and Steel Rat high quality models.
```

Comment:
This part was originally written with Arma 2 in mind. It should probably be split into one for Arma 2 and one for Arma 3 / current